cli/kaos_cli/facades/backend_facade.py

```diff
@@ -124,6 +210,44 @@ def _tf_init(self, provider, env, local_backend, destroying=False):
             self.tf_service.init(directory)
         return directory
 
+    def _set_context_list(self, current_context):
+        try:
+            available_contexts = self.state_service.get(CONTEXTS, 'environments')
+        except KeyError:
+            available_contexts = ''
+            
+        updated_contexts = available_contexts + current_context if not available_contexts else \
+            available_contexts + ',' + current_context
```

Comment:
I would expand this to a multi-line if-else statement for readability, it's almost two lines of the same length atm. Also, F-strings are handier here than `+` operations, it will also help your fellow engineers understand that the inferred type is always a string, i.e.:

```python
if available_contexts:
    updated_contexts = f"{available_contexts},{current_context}"
else:
    updated_contexts = current_context
```

Reply:
This has been re-written to handle multiple scenarios now. Also f string has been included